clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2009,6 +2009,16 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
   }];
 }
 
+def SYCLIntelUsesGlobalWorkOffsetDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "uses_global_work_offset (IntelFPGA)";
+  let Content = [{
+Applies to a device function/lambda function or function call operator (of a
+function object). If 0, ``get_global_offset(X)`` may not be called by the
```

Comment:
> If 0, ``get_global_offset(X)`` may not be called
I don't like this wording, ``get_global_offset(X)`` may not be called any way. Also I have a feeling, that get_global_offset() is OpenCL's build-in and not SYCL's.

Reply:
I suggest to replace this with:
```suggestion
function object). If 0, compiler doesn't use the global work offset values for the device function.
```